src/Components/v2/Lightbox/Lightbox.tsx

```diff
@@ -335,3 +348,5 @@ export class Lightbox extends React.Component<LightboxProps, LightboxState> {
     this.detectActivity()
   }
 }
+
+export const Lightbox = withSystemContext(LightboxComponent)
```

Comment:
This `withSystemContext` doesn't bother me, because `Lightbox` wasn't ever relying on `user` to be passed in.